introduction.md

```diff
@@ -1,2 +1,7 @@
 Hi, I'm esloho.
 My favorite Code School path is the Git Path but since that is not one of the options I'd say... JavaScript.
+
+* Git
+* Java
+* JavaScript
+* CSS
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?